core/app/models/spree/order_updater.rb

```diff
@@ -186,7 +186,7 @@ def update_item_total
     end
 
     def persist_totals
-      order.save!(validate: false)
+      order.save!(validate: Spree::Config[:run_order_validations_on_order_updater])
```

Comment:
Out of curiosity, is there a reason for choosing the `[]` syntax here?

Reply:
You are right, the `.` version is the one used in the codebase. Thanks!